__tests__/app.test.js

```diff
@@ -31,8 +33,168 @@ describe("NCNews-Server Unit Tests", () => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 1`, () => {
+
+            return request(app).get(`/api/articles/1&#59;&nbsp;DROP&nbsp;TABLE&nbsp;articles`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 2`, () => {
+
+            return request(app).get(`/api/articles/1; DROP TABLE articles`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+
+    });
+
+     // Task 6 - GET /api/articles/:article_id/comments
+
+     describe(`GET /api/articles/:article_id/comments`, () => {
+
+        test(`[ 200 ] Responds with an object with key comments and keyval an array of comments, where each comment should have the following properties: comment_id, votes, created_at, author, body`, () => {
+
+            return request(app).get(`/api/articles/1/comments`).expect(200).then((response) => {
+                const comments = response.body.comments;
+
+                expect(comments).toBeInstanceOf(Array);
+                expect(comments).toHaveLength(11);
+
+                comments.forEach((commentObject) => {
+                    expect(commentObject).toEqual(
+                        expect.objectContaining({
+                            comment_id: expect.any(Number),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            author: expect.any(String),
+                            body: expect.any(String)
+                    }));
+                });
+            });
+        });
+
+        test(`[ 404 ] Responds with an error when passed invalid parameters`, () => {
+
+            return request(app).get(`/api/articles/sdfdefds/comments`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 1`, () => {
+
+            return request(app).get(`/api/articles/1&#59;&nbsp;DROP&nbsp;TABLE&nbsp;articles/comments`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 2`, () => {
+
+            return request(app).get(`/api/articles/1; DROP TABLE articles/comments`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
```

Comment:
You must also test the following:

- Status 200, valid ID, but has no comments responds with an empty array of comments

- Status 404, non existent ID, e.g. 0 or 9999